release-notes/8.0/supported-os.json

```diff
@@ -0,0 +1,345 @@
+                {
+                    "id": "fedora",
+                    "name": "Fedora",
+                    "link": "https://fedoraproject.org/",
+                    "lifecycle": "https://fedoraproject.org/wiki/End_of_life",
+                    "architectures": [
+                        "Arm32",
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "40",
+                        "39"
+                    ],
+                    "unsupported-versions": [
+                        "38"
+                    ]
+                },
+                {
+                    "id": "opensuse",
+                    "name": "openSUSE Leap",
+                    "link": "https://www.opensuse.org/",
+                    "lifecycle": "https://en.opensuse.org/Lifetime",
+                    "architectures": [
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "15.6",
+                        "15.5"
+                    ]
+                },
+                {
+                    "id": "rhel",
+                    "name": "Red Hat Enterprise Linux",
+                    "link": "https://access.redhat.com/",
+                    "lifecycle": "https://access.redhat.com/support/policy/updates/errata/",
+                    "architectures": [
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "9",
+                        "8"
+                    ],
+                    "notes": [
+                        "Red Hat family distributions are supported per [Linux compatibility and support](../../linux-support.md)."
+                    ]
+                },
+                {
+                    "id": "sles",
+                    "name": "SUSE Enterprise Linux",
+                    "link": "https://www.suse.com/",
+                    "lifecycle": "https://www.suse.com/lifecycle/",
+                    "architectures": [
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "15.5"
+                    ]
+                },
+                {
+                    "id": "ubuntu",
+                    "name": "Ubuntu",
+                    "link": "https://ubuntu.com/",
+                    "lifecycle": "https://wiki.ubuntu.com/Releases",
+                    "architectures": [
+                        "Arm32",
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "24.04",
+                        "22.04",
+                        "20.04"
+                    ],
+                    "unsupported-versions": [
+                        "23.10"
```

Comment:
This should be `23.04`. `23.10` is missing in the `supported-versions` section (still supported for 2-3 more weeks).

Reply:
Fixed.